src/CofreeBot/Bot/Calculator.hs

```diff
@@ -1,11 +1,11 @@
 module CofreeBot.Bot.Calculator where
 
-import CofreeBot.Bot
-import CofreeBot.Bot.Calculator.Language
-import CofreeBot.Utils
-import Data.Profunctor
-import Data.Text qualified as T
-import Data.Functor
+import           CofreeBot.Bot
+import           CofreeBot.Bot.Calculator.Language
+import           CofreeBot.Utils
+import           Data.Functor
+import           Data.Profunctor
+import qualified Data.Text                     as T
```

Comment:
Is it possible to keep `ImportQualifiedPost`? I don't like having the huge indents on the non qualified imports to maintain alignment.